pkg/agent/keys.go

```diff
@@ -0,0 +1,105 @@
+package agent
+
+import (
+	"crypto"
+	"fmt"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/agent/keys"
+	"go.etcd.io/bbolt"
+)
+
+type keyInt interface {
+	crypto.Signer
+	//Type() string // Not Yet Supported by Krypto
+}
+
+var Keys keyInt = keys.Noop
+var LocalDbKeys keyInt = keys.Noop
```

Comment:
mentioned in other comment, I would prefer to have these unexported and exposed an expose via a getter.

Reply:
(resolving this, because we only need one place to talk about it)